AUTHORS

```diff
@@ -181,7 +181,6 @@ Ryan Petrello <lists@ryanpetrello.com>
 Fuji Goro <gfuji@cpan.org>
 Siddharth Mahendraker <siddharth_mahen@hotmail.com>
 Dave Pacheco <dap@joyent.com>
-Mathias Buus <m@ge.tt>
```

Comment:
The existing entries should not be removed. These are roughly listed in the order of contribution. Removing the existing entries and moving them further down breaks that. If there are duplicates, then the *new* entries should be dropped.